scripts/unocommands.py

```diff
@@ -12,8 +12,10 @@
 import sys
 from lxml import etree
 
+
 def usageAndExit():
-    message = """usage: {program} [--check|--update|--translate] online_dir [...]
```

Comment:
I think this line needs to stay as it is. This is part of help text, telling user about the command-line options. Please revert the change on this line.